apps/api/src/admin/participant_manager.py

```diff
@@ -117,3 +117,13 @@ async def confirm_attendance_non_hacker(uid: str, director: User) -> None:
         raise RuntimeError(f"Could not update status to ATTENDING for {uid}.")
```

Comment:
`Role` is currently not exported from `useParticipants`, which causes an error. Selective staging?